code/__HELPERS/view.dm

```diff
@@ -1,15 +1,10 @@
 /proc/getviewsize(view)
-	var/viewX
-	var/viewY
 	if(isnum(view))
-		var/totalviewrange = (view < 0 ? -1 : 1) + 2 * view
-		viewX = totalviewrange
-		viewY = totalviewrange
+		var/totalviewrange = 1 + 2 * view
```

Comment:
The ternary statement was here for a reason. I don't remember what reason, but there was one

Reply:
copypasta will fix

Reply:
Done